examples/blog-with-auth/apps/admin/.env.development

```
@@ -1,6 +1,6 @@
 VITE_ENVIRONMENT=dev
 VITE_GRAPH_API_ENDPOINT=/api/graphql
 VITE_SENTRY_DSN=
-DEV_BACKEND_HOST=http://localhost:3001
+DEV_BACKEND_HOST=http://localhost:5001
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DEV_BACKEND_HOST key should go before the VITE_ENVIRONMENT key

(UnorderedKey)